setup.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
I'm not sure whether a shebang like this even makes sense here, since you're always running `python setup.py install` anyway?